app/javascript/packs/Events/event.jsx

```diff
@@ -198,13 +229,19 @@ export default class Event extends React.Component {
               {checkbox}
             </Col>
             <Col
-              md={9}
+              md={7}
               as='h5'
               style={{ overflow: "hidden" }}
             >
               {this.props.title}
             </Col>
-            <Col md={2}>
+            <Col
+              md={2}
+              as='h3'
+            >
+              {ProcessButton}
```

Comment:
Your button moves checkbox out of list group item.

Reply:
Solved